lib/gnat/jetstream/api/stream.ex

```diff
@@ -441,6 +441,17 @@ defmodule Gnat.Jetstream.API.Stream do
     end
   end
 
+  @doc """
+  Returns true if the provided stream is a KV bucket, false otherwise
+
+  ## Parameters
+  * `stream_name` - the stream name to test
+  """
+  @spec is_kv_bucket_stream(stream_name :: binary()) :: boolean()
+  def is_kv_bucket_stream(stream_name) do
```

Comment:
I would suggest moving this function to the `KV` module. So far I've been keeping all of the KV-specific knowledge in that module and similarly keeping everything related to `Object` in its own module, so the stream module really just has knowledge of general streams.

I'm definitely open to feedback here if you had a specific reason for putting it into the stream module.

Reply:
No specific reason other than the fact that it's dealing with the actual stream.
Can definitely be moved to the KV module